pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -122,6 +122,20 @@ type TresorCASpec struct {
 // can be (Active, Passive).
 type MeshRootCertificateIntent string
 
+const (
+	// ActiveIntent
+	ActiveIntent MeshRootCertificateIntent = "active"
+
+	// PassiveIntent
+	PassiveIntent MeshRootCertificateIntent = "passive"
+
+	// DeactiveIntent
+	DeactiveIntent MeshRootCertificateIntent = "deactive"
+
+	// InactiveIntent
+	InactiveIntent MeshRootCertificateIntent = "inactive"
```

Comment:
Why is there an intent for both Inactive and Deactivate? Why would the user set one instead of the other?

Reply:
This makes the transition from singing -> validation for the Active cert and from validating -> signing for the Passive cert deterministic.  I made a comment here as well: https://github.com/openservicemesh/osm/pull/5201#discussion_r995157614
where I think we might be able to remove this and still make it deterministic most of the time which would simplify somethings. Be curious what you think.

Reply:
Yeah, if the user is setting these intents, having both (at least the way they're currently named) feels confusing.Maybe `Idle` is a better name for "Inactive" assuming I understand the intended usage correctly.